frontend/src/components/ScheduledEventsTable/ScheduledEventsTable.tsx

```diff
@@ -13,10 +13,12 @@ export const ScheduledEventsTable: FunctionComponent<ScheduledEventTableProps> =
 
   const renderRows = (): ReactNode =>
     props.events.map((event: ScheduledEvent, index: number) => {
+
```

Comment:
Remove blank line ... maybe even remove the `return` itself :)

Reply:
Removed the blank line, it breaks without `return` :)

Reply:
Hehe ... ofc the structure has to change without the return ... not required though, so never mind....